portal/portal-impl/impl/src/bundle/sitenav.properties

```diff
@@ -115,13 +115,13 @@ pda_back_to_sites = Back to sites list
 pda_subsite_lead = --  
 pda_subsite = (Sub Site)
 
-join_title = Joinable Site
-join_instr = You are not currently a member of the site:
-join_question = Would you like to join the site?
+join_title = Join this Site
+join_instr = Currently, you are not a member of the worksite called "{0}".
+join_question = Would you like to join and become a member of the worksite?
 join_restricted = This site has limited access. You cannot join the site with your current account. Contact the site's owner for more information.
 
-yes = Yes
-no = No
+yes = Yes, please add me
+no = No thanks
```

Comment:
not a huge fan of please and thanks in UI .... happy to listen to opinions from others. 

I see one other use of "No, thanks" in the import ui in site-manage.

I'd prefer direct words here instead.... like Yes/No or "Join Site" and "Do Not Join" or "No"

Reply:
Yes, I agree on the consistency point: we don't have "please" and "thanks" in other areas. Maybe we'll add it to the Canadian English translations some day. :)

Therefore, I propose: "Yes, add me" and "No, I do not want to join" instead.

Reply:
The reason we wanted more text than simple "Yes" and "No" for the button text on this form is the simplicity and bareness of the page doesn't provide much context to people coming to this page - particularly if it is linked from a non-Sakai site.

Reply:
Yeah the reason makes sense to me .... I just would remove Please/Thanks.... Thanks ;)

https://ifunny.co/fun/GPMpYLbE46